projects/plutus/src/ltc_adc_calibration.c

```diff
@@ -0,0 +1,29 @@
+#include "ltc_adc_calibration.h"
+
+static void prv_callback(int32_t *value, void *context) {
```

Comment:
Please consider how we can use the current reading. We do not poke into internal storage as a rule of thumb.

Reply:
Fixed